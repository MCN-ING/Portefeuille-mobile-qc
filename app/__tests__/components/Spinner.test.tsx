import { render } from '@testing-library/react-native'
import Spinner from '../../src/components/Spinner'

describe('Spinner Component', () => {
  test('Renders correctly', () => {
    const tree = render(<Spinner />)

    expect(tree).toMatchSnapshot()
  })
})
