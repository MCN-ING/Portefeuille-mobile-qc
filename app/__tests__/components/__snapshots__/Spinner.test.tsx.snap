// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner Component Renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "justifyContent": "center",
    }
  }
  testID="com.ariesbifold:id/LoadingActivityIndicator"
>
  <View
    collapsable={false}
    style={
      {
        "position": "absolute",
        "transform": [
          {
            "rotate": "0deg",
          },
        ],
      }
    }
  >
    <
      height={85}
      width={85}
    />
  </View>
</View>
`;
